packages/wonder-stuff-testing/src/jest/wait.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import {verifyRealTimers} from "./internal/verify-real-timers.js";
+import {unverifiedWait} from "./internal/unverified-wait.js";
+import {assertJest} from "./internal/assert-jest.js";
+
+assertJest();
```

Comment:
Hmm, I wonder if after the rollup, this will do what I expect. I'm guessing not. Should probably move this to inside the various calls so it's asserted at the latest point necessary.

Reply:
OK, I've modified this.